slang.h

```diff
@@ -4134,6 +4134,26 @@ namespace slang
         None, UnsizedArray, StructuredBuffer, ConstantBuffer, ParameterBlock
     };
 
+    // A struct for storing checksums designed to be used with the MD5 hashing implementation
+    struct Checksum
+    {
+        uint32_t checksum[4] = { 0 };
+
+        Checksum& operator=(const Checksum& rhs)
```

Comment:
Why is this necessary? We should be able to assign directly without this overload.

Reply:
I added it because you mentioned it on Slack yesterday, mostly. I'll just remove it if direct assignment works fine

Reply:
Sorry, I meant `operator ==` yesterday.

Reply:
yeah looks fine, getting rid of this